### DISCUSSION TITLE
Renaming "Qwik City" into "Qwik Router" in V2

### DISCUSSION THREAD

**Discussion by shairez** [QwikDev/qwik-evolution#185] [Proposals For Qwik]
#### Problems you are trying to solve:
* "Qwik City" as a name requires extra explanation
* A meta framework on top of Qwik (which is already very different than any other framework) adds another layer of confusion when learning about it

#### Goals you are trying to achieve:
* Simplicity in both learning and explaining about Qwik

#### Any other context or information you want to share:

---

#### What do you propose?

In V2, the plan is to have the only "breaking changes" be the packages renaming.

Qwik will become `@qwik.dev/core`
and Qwik City will become `@qwik.dev/router` 

99% of the stuff inside of "Qwik City" is routing related - even SSG, loaders and actions and server functions (which are RPCs "routed" to the right place on the server)

So in the end of the day, renaming this package with no breaking API changes (everything will stay backwards compatible with Qwik City) we think will make things easier for Qwik developers.

**Comment by PatrickJS** [comment]
Yes let’s change it

**Comment by steffanek** [comment]
Totally agree!!

**Comment by kolinfluence** [comment]
when is v2 coming?

**Reply by shairez** [reply]
soon alpha will be released

**Comment by shairez** [comment]
this proposal is tracked under #186 
closing this discussion.
